doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,12 @@
 
 <h3>New features since last release</h3>
 
+* Functions are added for generating spin Hamiltonians for Emery and Haldane
+  models on a lattice.
+  * `qml.spin.emery` creates the [Emery model](https://arxiv.org/pdf/2309.11786) Hamiltonian.
+    * `qml.spin.haldane` creates the [Haldane model](https://arxiv.org/pdf/2211.13615) Hamiltonian.
```

Comment:
Links could be added in the main sentence itself to make the changelog more succinct.